tsconfig.js.json

```diff
@@ -0,0 +1,8 @@
+{
+	"extends": "./tsconfig.release.json",
+	"compilerOptions": {
+		"noImplicitAny": false,
```

Comment:
Without this the js files have a lot of implicit any errors which makes sense and isn't something we're interested in